### DISCUSSION TITLE
No package 'yes' found magick package

### DISCUSSION THREAD

**Discussion by dvargas135** [libvips/libvips#2449] [Q&A] [answered]
I used `./configure --with-magickpackage` and then checked the config log. I happened to find this error but I'm not sure how to install 'yes' package

```
configure:17749: $PKG_CONFIG --exists --print-errors "$with_magickpackage"
Package yes was not found in the pkg-config search path.
Perhaps you should add the directory containing `yes.pc'
to the PKG_CONFIG_PATH environment variable
No package 'yes' found
configure:17752: $? = 1
configure:17766: result: no
No package 'yes' found
configure:17782: WARNING: yes not found; disabling Magick support
```

**Comment by jcupitt** [comment, marked as answer]
Hi @dvargas135,

You need to give the name of the magick package you want to use, for example:

```
./configure --with-magickpackage=GraphicsMagick
```

By default it'll use ImageMagick, so if it wasn't detecting it, you're probably missing the development headers.

**Comment by dvargas135** [comment]
sorry for opening another issue. I couldn't comment on this one because it was closed

----

I built imagemagick with the flag --with-modules and everything went fine but when i tried to build libvips afterwards i get the same error

```
Package yes was not found in the pkg-config search path.
Perhaps you should add the directory containing `yes.pc'
to the PKG_CONFIG_PATH environment variable
```
I would like to use imagemagick and not graphicsmagick
I'm using ImageMagick 7.1.0-8 Q16-HDRI x86_64 2021-09-18 and vips 8.11.4

Where can i find this 'yes' package?

**Reply by jcupitt** [reply]
The libvips configure `--with-magickpackage` flag needs an argument: the name of the magick package you want to build against:

```
./configure --with-magickpackage=GraphicsMagick
```

By default it'll search for ImageMagick, so in your case just:

```
./configure 
```

should work.

**Reply by jcupitt** [reply]
I added an error message to libvips master if the argument is missing.

**Comment by dvargas135** [comment]
Thank you, worked like a charm.
